generated/python/gapic-google-cloud-logging-v2/google/cloud/gapic/logging/v2/logging_service_v2_client.py

```diff
@@ -49,15 +52,17 @@ class LoggingServiceV2Client(object):
     DEFAULT_SERVICE_PORT = 443
     """The default port of the service."""
 
-    _CODE_GEN_NAME_VERSION = 'gapic/0.1.0'
-
-    _GAX_VERSION = pkg_resources.get_distribution('google-gax').version
-
     _PAGE_DESCRIPTORS = {
         'list_log_entries': _PageDesc('page_token', 'next_page_token',
                                       'entries'),
         'list_monitored_resource_descriptors':
-        _PageDesc('page_token', 'next_page_token', 'resource_descriptors')
+        _PageDesc('page_token', 'next_page_token', 'resource_descriptors'),
+        'list_logs': _PageDesc('page_token', 'next_page_token', 'log_names')
+    }
+
+    _BUNDLE_DESCRIPTORS = {
```

Comment:
@bjwatson @lukesneeringer 
I see that @garrettjonesgoogle added the bundling config back to the GAPIC config for logging a few days ago, resulting in this diff. However, I'm not sure we want to expose bundling on the Python surface given that it's not a feature we maintain or promote. What are your thoughts here?

If we don't want to expose bundling on any GAPIC surface for the time being, we should go ahead and remove this code from the Python GAPIC templates in `toolkit`.